packages/xo-web/src/xo-app/jobs/new/index.js

```diff
@@ -440,64 +507,15 @@ export default class Jobs extends Component {
             </fieldset>
           )}
         </form>
-        <table className='table'>
-          <thead>
-            <tr>
-              <th>{_('jobName')}</th>
-              <th>{_('jobAction')}</th>
-              <th />
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            {isEmpty(jobs) && (
-              <tr>
-                <td>
-                  <em>{_('noJobs')}</em>
-                </td>
-              </tr>
-            )}
-            {map(jobs, job => (
-              <tr key={job.id}>
-                <td>
-                  <span>
-                    {job.name}{' '}
-                    <span className='text-muted'>({job.id.slice(4, 8)})</span>
-                  </span>
-                </td>
-                <td>{job.method}</td>
-                <td>
-                  <ActionRowButton
-                    disabled={!isJobUserMissing[job.id]}
-                    icon='run-schedule'
-                    btnStyle='warning'
-                    handler={runJob}
-                    handlerParam={job.id}
-                  />
-                  {!isJobUserMissing[job.id] && (
-                    <Tooltip content={_('jobUserNotFound')}>
-                      <Icon className='ml-1' icon='error' />
-                    </Tooltip>
-                  )}
-                </td>
-                <td>
-                  <ActionRowButton
-                    icon='edit'
-                    btnStyle='primary'
-                    handler={this._edit}
-                    handlerParam={job.id}
-                  />{' '}
-                  <ActionRowButton
-                    icon='delete'
-                    btnStyle='danger'
-                    handler={deleteJob}
-                    handlerParam={job.id}
-                  />
-                </td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
+        <SortedTable
+          collection={jobs}
+          columns={COLUMNS}
+          groupedActions={GROUPED_ACTIONS}
+          individualActions={this.individualActions}
+          shortcutsTarget='body'
+          stateUrlParam='s'
+          userData={{ isJobUserMissing: this._getIsJobUserMissing() }}
```

Comment:
You're building a new object on each render which will trigger a new render of the `SortedTable`. If there's only one field in `userData`, pass it directly.